src/pages/EventList/index.tsx

```diff
@@ -17,7 +17,7 @@ import EventResource from '../../resources/event';
 import LocationResource from '../../resources/location';
 import EventsListItem from './EventListItem';
 
-const dateFormat = 'dddd, MMMM D';
+const dateFormat = 'dddd, MMM D';
```

Comment:
Is this an intended change to the date format?

Reply:
assuming this means using the 3-letter abbreviation of months rather than a 4-letter abbreviation, I'm in favor of this change

Reply:
yes, intended